microsite/data/plugins/end-of-life.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: End of life
+author: dweber019
```

Comment:
Optional: You can use your full name here if you prefer.

Reply:
I think I keep it like this to have it consistent any way thinking about a mono repo for all my plugins so I could change it then 😉